src/services/ArticleService.js

```diff
@@ -19,21 +31,14 @@ export class ArticleService {
 	async getArticleById(id) {
 		try {
 			const { data } = await axios.get(`${backendAPI}/${this.BASE_URL}/${id}`);
-			const [response] = data.response;
-			const { article } = response;
+			const [article] = data.response;
 
 			const date = new Date(+article.createdOn).toString().split(" ");
 
 			return {
-				id: id,
-				title: article.title,
-				description: article.description,
-				author: {
-					id: article.createdBy.id,
-					name: article.createdBy.alias
-				},
-				content: article.content,
-				created: `${date[2]} ${date[1]} ${date[3]}`
+				...article,
+				createdOn: `${date[2]} ${date[1]} ${date[3]}`,
+				path: ArticleService.buildArticleURL(article)
```

Comment:
I would adjust the dates to use locale specific formatting
`new Date().toLocaleDateString(navigator.language, {year: 'numeric', month: 'long', day: 'numeric'});`